bootset/bootset

```diff
@@ -8,7 +8,7 @@
 # ╚═════╝ ╚══════╝ ╚═════╝ ╚══════╝╚═════╝ ╚═╝  ╚═╝╚═╝  ╚═══╝ ╚══▀▀═╝  ╚═════╝ ╚═╝╚══════╝╚══════╝
 #
 # booset tool, to manage nodes PXE boot
-# 2019_2020 - Benoît Leveugle <benoit.leveugle@sphenisc.com>
+# 2019_2021 - Benoît Leveugle <benoit.leveugle@sphenisc.com>
```

Comment:
I don't think we should keep the dates in the tool. This is not a changelog, so this is not a valuable data.

Reply:
OK. Updated using users registered in github history on this file at https://github.com/bluebanquise/bluebanquise/blob/master/roles/core/pxe_stack/files/bootset.py

Reply:
Perfect ! Bruno's name is Travouillon. Once this is done, ok for me to merge !